boards/arm/thingy52_nrf52832/board.c

```diff
@@ -18,7 +18,7 @@ struct pwr_ctrl_cfg {
 
```

Comment:
In the commit message better grep command would be
```
git grep -rlz 'dev)->config_info' |
```
as that would skip untracked files. This would not change the outcome of this commit, but if I would try this on my zephyr tree, there would be file corruption on some of the untracked files (GTAGS in this case).